src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -94,43 +95,35 @@ def test_Enum_attribute(device_info):
         assert session.function == nidmm.Function.AC_CURRENT
         assert type(session.function) is nidmm.Function
         try:
-            session.function = nidmm.LCCalculationModel.CALC_MODEL_SERIES
+            session.function = nidmm.LCCalculationModel.SERIES
             assert false
         except TypeError as e:
             print(e)
             pass
 
 
-def test_ViSession_attribute(device_info):
+def test_acquisition(device_info):
     with nidmm.Session(device_info['name']) as session:
-        try:
-            session.io_session = 5
-            assert false
-        except TypeError as e:
-            print(e)
-            pass
-        try:
-            value = session.io_session
-            assert false
-        except TypeError as e:
-            print(e)
-            pass
-
+        session.configure_measurement_digits(nidmm.Function.DC_CURRENT, 1, 5.5)
+        with session.initiate():
+            print(session.fetch(1000))
+        with session.initiate():
+            print(session.fetch(1000))
 
-def test_parameter_count0(device_info):
+			
+def test_function_call_with_zero_parameter(device_info):
     with nidmm.Session(device_info['name']) as session:
         assert session.get_aperture_time_info()[1] == 0 # Assuming default aperture time unit will be seconds
-
-                
-def test_parameter_count1(device_info):
+   
+        
+def test_function_call_with_one_parameter(device_info):
     with nidmm.Session(device_info['name']) as session:
         session.configure_measurement_digits(nidmm.Function.DC_VOLTS, 1, 5.5)
```

Comment:
The test is test_function_call_with_one_parameter but in the body a lot of stuff is happening.
Can you pare it down? For example, a lot happens when we call initiate().

There must be some 1 param method you can use.